pkgs/applications/networking/instant-messengers/chatterino7/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, cmake, pkg-config, fetchFromGitHub, qtbase, qtsvg, qtmultimedia, qtimageformats, qttools, boost, openssl, wrapQtAppsHook, libsecret }:
+
+stdenv.mkDerivation rec {
+  pname = "chatterino7";
+  version = "7.4.5";
+  src = fetchFromGitHub {
+    owner = "SevenTV";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-wLiGnNVr1n6LX6bzjqUrk99K981Kz2Mqc2ydU94t4l0=";
+    fetchSubmodules = true;
+  };
+  nativeBuildInputs = [ cmake pkg-config wrapQtAppsHook ];
+  buildInputs = [ qtbase qtsvg qtmultimedia qtimageformats qttools boost openssl libsecret ];
+
+  postInstall = lib.optionalString stdenv.isDarwin ''
+    mkdir -p "$out/Applications"
+    mv bin/chatterino.app "$out/Applications/"
+  '' + ''
+    mkdir -p $out/share/icons/hicolor/256x256/apps
+    cp $src/resources/icon.png $out/share/icons/hicolor/256x256/apps/chatterino.png
```

Comment:
Can you try this without `$src/` ? That should already be unpacked into the current directory and that makes working with nix-shell easier.

Reply:
no, you can just leave it as is then